backend/services/mgmt/v1alpha1/transformers-service/transformers.go

```diff
@@ -516,6 +517,17 @@ func (s *Service) GetSystemTransformers(
 					},
 				},
 			},
+			{
+				Name:        "Javascript",
+				Description: "Write custom javascript to transform data",
+				DataType:    "string",
```

Comment:
Not really sure this data type makes sense here. The javascript transformer/generator should work with any data type.

Reply:
updating to "any"